src/client.ts

```diff
@@ -1961,35 +1964,68 @@ export class StreamChat<StreamChatGenerics extends ExtendableGenerics = DefaultG
    *
    * @param {string[]} user_ids which users to restore
    *
-   * @return {APIResponse} A task ID
+   * @return {APIResponse} An API response
    */
   async restoreUsers(user_ids: string[]) {
     return await this.post<APIResponse>(this.baseURL + `/users/restore`, {
       user_ids,
     });
   }
 
-  async reactivateUser(
-    userID: string,
-    options?: {
-      created_by_id?: string;
-      name?: string;
-      restore_messages?: boolean;
-    },
-  ) {
+  /**
+   * reactivateUser - Reactivate one user
+   *
+   * @param {string} userID which user to reactivate
+   * @param {ReactivateUserOptions} [options]
+   *
+   * @return {UserResponse} Reactivated user
+   */
+  async reactivateUser(userID: string, options?: ReactivateUserOptions) {
     return await this.post<APIResponse & { user: UserResponse<StreamChatGenerics> }>(
       this.baseURL + `/users/${userID}/reactivate`,
       { ...options },
     );
   }
 
-  async deactivateUser(userID: string, options?: { created_by_id?: string; mark_messages_deleted?: boolean }) {
+  /**
+   * reactivateUsers - Reactivate many users asynchronously
+   *
+   * @param {string[]} user_ids which users to reactivate
+   * @param {ReactivateUsersOptions} [options]
+   *
+   * @return {TaskResponse} A task ID
+   */
+  async reactivateUsers(user_ids: string[], options?: ReactivateUsersOptions) {
+    return await this.post<APIResponse & TaskResponse>(this.baseURL + `/users/reactivate`, { user_ids, ...options });
+  }
+
+  /**
+   * deactivateUser - Deactivate one user
+   *
+   * @param {string} userID which user to deactivate
+   * @param {DeactivateUsersOptions} [options]
+   *
+   * @return {UserResponse} Deactivated user
+   */
+  async deactivateUser(userID: string, options?: DeactivateUsersOptions) {
     return await this.post<APIResponse & { user: UserResponse<StreamChatGenerics> }>(
       this.baseURL + `/users/${userID}/deactivate`,
       { ...options },
     );
   }
 
+  /**
+   * deactivateUsers - Deactivate many users asynchronously
+   *
+   * @param {string[]} user_ids which users to deactivate
+   * @param {DeactivateUsersOptions} [options]
+   *
+   * @return {TaskResponse} A task ID
+   */
+  async deactivateUsers(user_ids: string[], options?: DeactivateUsersOptions) {
```

Comment:
nit: to align with others, might want to rename it to `userIDs` and internally do `user_ids: userIDs`

Reply:
if you look at other endpoints, we use `user_ids`

Reply:
my point is difference between single vs batch ones

Reply:
yeah but since others use `user_ids`, i don't see the real value honestly

Reply:
value is clear, mixing casing is unprofessiona and not user intuitive but should be fixed globally